py3status/modules/rss.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+"""
+Display the unread feed items in your OwnCloud/NextCloud account.
+
+Configuration parameters:
+    aggregator: feed aggregator used, see note below (default 'owncloud')
+    cache_timeout: how often to run this check (default 60)
+    feeds_id: list of IDs of feeds to watch, see note below (default [])
+    folders_id: list of IDs of folders ro watch, see note below (default [])
+    format: format to display (default 'Feed: {unseen}')
+    hide_if_zero: don't show on bar if False (default False)
+    new_items_color: text color when counter isn't zero (default '')
+    password: login password (default '<PASSWORD>')
+    server: IMAP server to connect to (default 'https://yourcloudinstance.com')
+    user: login user (default '<USERNAME>')
+
+Format placeholders:
+    {unseen} sum of numbers of unread feed elements
+
+For now, the only supported value of `aggregator` is `'owncloud'`, that means that you'll
+only be able to configure a OwnCloud/NextCloud instance. Other feed aggregator may be
+supported in future releases.
+
+`feeds_id` and `folders_id` enable to restrict feed items counting to the given feeds and
+folders IDs. You can get those ID by clicking on the desired feed or folder and watching
+the URL:
+```
+https://yourcloudinstance.com/index.php/apps/news/#/items/feeds/FEED_ID
+https://yourcloudinstance.com/index.php/apps/news/#/items/folders/FOLDER_ID
+
+```
+If both are left empty, all unread feed items will be counted. Counted items don't need
+to match both feeds and folders lists. Items matching both a given feed ID and a given
+folder ID will be counted only once.
```

Comment:
This commentary is in the wrong place in the docstring. And Could at least link to the owncloud github page or similar.

Sections for colors and requirements are missing.